### PR TITLE
Fix bibtex entries for ASPECT.

### PR DESCRIPTION
**Pull request by njinju**
There are several bibtex entries issues in the tree for ASPECT where some necessary capital letters such as in  "Advanced Solver for Problems in Earth ConvecTion" or in names like "Stokes" turns out to show small letters when bibtex runs. We fix this issue by enclosing all necessary capital letters into curly braces.

### CODE REVIEW

File: doc/manual/manual.bib [hunk 1]
```diff
@@ -11315,7 +11315,7 @@ @misc{aspect-doi-v2.0.0
 }
 
 @article{aspectmanual,
-  title         = {{\textsc{ASPECT}: Advanced Solver for Problems in Earth's ConvecTion, User Manual}},
+  title         = {{\textsc{ASPECT}: {A}dvanced {S}olver for {P}roblems in {E}arth's {C}onvec{T}ion, User Manual}},
```
Comment (tjhei) [inline doc/manual/manual.bib]: this change is not necessary as there are already braces around the whole title.

**Comment by tjhei** [comment]
related to #1097

**Comment by tjhei** [comment]
@bangerth it this what you had in mind? Another option is to just put the whole title in braces.

File: doc/manual/manual.bib (hunk 1 above)
Comment (bangerth) [reply doc/manual/manual.bib]: True, but there's also no harm here.

**Comment by bangerth** [comment]
Yes, that's what I thought should be done.

Thanks, @njinju !

**Pull request merged**